static/docs/user-guide/howto/keep-data-and-cache-on-external-drive.md

```diff
@@ -0,0 +1,200 @@
+# Keep Data and Cache on External Drive
+
+Sometimes the data may be stored on an
+[external hard drive](https://whatis.techtarget.com/definition/external-hard-drive).
+Usually such data is huge, which means that it won't fit on our local drive, and
```

Comment:
This motivation is not enough. Do you remember where did this case come from? It's not clear why do I need two drives, what advantages does this configuration has if I store everything - input data, results, etc outside? I understand the case when we do this with S3, for example.

Reply:
> Do you remember where did this case come from?

Normally I would not remember, however the issue #563 has some context about it. A user has requested help for this situation on the chat. I am not sure why he has to keep the data on an external drive, but keep the project on his computer. But this is how he prefers to organize his work.

Recently I have seen another discussion where a user wants to keep all the data and cache on an external directory, on a fixed absolute path, so that he can search easily the code of the project. Again I am not sure whether this is the most efficient solution for him, but this is what he does. (Sorry, I cannot find a reference for this discussion.)
I would say that the external-data case is very similar to the external-drive case.

> I understand the case when we do this with S3, for example.

I am not sure about the benefits of using this approach for the S3 case, because the data would still have to be downloaded, in order to be processed locally, and it would still have to be uploaded, in order to be saved remotely. The benefits are not so clear, in my opinion. You might be able to save some local storage space, but this comes at the cost of increased downloads and uploads.